queue/execution.go

```diff
@@ -87,3 +112,11 @@ func (eq *ExecutionQueue) GetRepositories() []string {
 	}
 	return repositories
 }
+
+func (eq *ExecutionQueue) incQueueCount(namespace, jobType string) {
+	queueCount.WithLabelValues(namespace, jobType).Inc()
+}
+
+func (eq *ExecutionQueue) decQueueCount(namespace, jobType string) {
+	queueCount.WithLabelValues(namespace, jobType).Dec()
+}
```

Comment:
Prometheus makes clear distinction between counters (not decrementable) and gauges. Let's not mix them in the naming :)
```suggestion
func (eq *ExecutionQueue) incQueueGauge(namespace, jobType string) {
	queueGauge.WithLabelValues(namespace, jobType).Inc()
}

func (eq *ExecutionQueue) decQueueGauge(namespace, jobType string) {
	queueGauge.WithLabelValues(namespace, jobType).Dec()
}
```